src/App.js

```diff
@@ -1,39 +1,60 @@
 import PeriodicTable from './Components/PeriodicTable.js';
 import styled from 'styled-components';
+import {Icon} from '@iconify/react';
```

Comment:
If you do not use a variable you should delete it.

Reply:
thanks, i left it there because I want to use Icons from iconify at some point.